tests/conftest.py

```diff
@@ -19,9 +21,15 @@ def session_fixture():
 def client_fixture(session: Session):
     def get_session_override():
         return session
+    
+    def override_get_current_user():
+        # Return a dummy user or None to bypass authentication
+        return Users(username="testuser", hashed_password="dummyhash")
 
     case_api.dependency_overrides[get_session] = get_session_override
 
+    case_api.dependency_overrides[get_current_user] = override_get_current_user
```

Comment:
Can we update the testing documentation to include the fact that this client mocks the authentication please.